src/contacts/ViewContact/ContactAddresses/ContactAddress.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { AddressView } from '@folio/stripes/smart-components';
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+const visibleFields = [
+  'addressLine1',
+  'addressLine2',
+  'city',
+  'stateRegion',
+  'zipCode',
+  'country',
+  'language',
+  'categories',
+];
+
+const labelMap = {
+  addressLine1: <FormattedMessage id="ui-organizations.contactPeople.addressLine1" />,
+  addressLine2: <FormattedMessage id="ui-organizations.contactPeople.addressLine2" />,
+  city: <FormattedMessage id="ui-organizations.contactPeople.city" />,
+  stateRegion: <FormattedMessage id="ui-organizations.contactPeople.stateRegion" />,
+  zipCode: <FormattedMessage id="ui-organizations.contactPeople.zipCode" />,
+  country: <FormattedMessage id="ui-organizations.contactPeople.country" />,
+  language: <FormattedMessage id="ui-organizations.contactPeople.language" />,
+  categories: <FormattedMessage id="ui-organizations.contactPeople.categories" />,
+};
+
+const ContactAddress = ({ address }) => (
+  <Row>
+    <Col xs={12}>
+      <AddressView
+        addressObject={address}
+        visibleFields={visibleFields}
+        labelMap={labelMap}
+      />
+    </Col>
+  </Row>
+);
+
+ContactAddress.propTypes = {
+  address: PropTypes.object.isRequired,
+};
+
+export default ContactAddress;
```

Comment:
What is the reason of this new component? the old one looks the same, no?

Reply:
Yes. It is placed in `ContactPeople` folder and has a title. But now this title not needed because of accordion title usage.

Reply:
Shouldn't we just change that component instead of creating a new one?

Reply:
We use both. One in `ContactInformation`, another one in `ViewContact`

Reply:
Are they should be different? If previously they were the same

Reply:
I don't know. But anyway it could be reusable if they are the same on new mockups